### PR TITLE
Resolves issue #100 (systemd script for debian/raspbian jessie)

### PR DESCRIPTION
**Pull request by Sufi-Al-Hussaini**
Hope I've placed the script in the right place.

### CODE REVIEW

File: dist-scripts/debian/gmrender-resurrect.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=gmrender-resurrect service
+After=network.target sound.target
+
+[Service]
+Environment="UPNP_DEVICE_NAME=Raspberry"
+Environment="UPNP_UUID=671ba82bb2add32f128675e011c453d0"
```
Comment (hzeller) [inline dist-scripts/debian/gmrender-resurrect.service]: Can you make the UUID dynamic as it is in the init.d script ? The problem with a fixed UUID is, that then all gmrender-resurrect on the same network can not be distinguished.

So if they are based on the MAC address or something (something that is predictable after reboot), that would be great.

**Comment by Sufi-Al-Hussaini** [comment]
Sorry for the late reply. I was a little caught up with another project on my blog. 

You're right about the fixed UUID. I'll fix this soon.

**Comment by Sufi-Al-Hussaini** [comment]
Now, I'm doing it the same way you have, in the `scripts/init.d/gmediarenderer` init script. 
I just realised bash substitution isn't that straightforward in systemd unit files. :) But I managed to get this done somehow.

**Comment by coldtobi** [comment]
bash is usually quite heavy? Would it also work with standard /bin/sh?

**Comment by Sufi-Al-Hussaini** [comment]
@coldtobi  
I'm not really a shell guy, but I just tried on my ubuntu desktop and it seems to be working. 👍

**Comment by hzeller** [comment]
Thanks for the changes. I'll merge it today and replace the bash with sh.

Maybe I should make it easier to create a st able UUID from the ethernet interface already in gmrender resurrect, but unfortunately, that would be system dependent (I know something that would work on Linux but not necessarily on other Unixes; though I presume this is mostly used on Linux anyway, so not really a big deal).

**Comment by Sufi-Al-Hussaini** [comment]
You're welcome. :)
I just replaced `bash` with `sh` myself.

> Maybe I should make it easier to create a st able UUID from the ethernet interface already in gmrender resurrect
That'll be great. 

And thanks for gmrender-resurrect!

**Comment by hzeller** [comment]
Thanks! Just pushed as https://github.com/hzeller/gmrender-resurrect/commit/57dfbfd474f9008cc91f7a57b38ec4a7f96ae593

**Pull request closed without merging**